src/basics.php

```diff
@@ -51,7 +51,13 @@ function debug($var, $showHtml = null, $showFrom = true)
         $line = '';
         $lineInfo = '';
         if ($showFrom) {
-            $trace = Debugger::trace(['start' => 1, 'depth' => 2, 'format' => 'array']);
+            $trace = Debugger::trace(['start' => 1, 'depth' => 3, 'format' => 'array']);
+            if (count($trace) > 1
+                && $trace[1]['function'] === 'dd'
```

Comment:
Having a method check for another method (dd) which calls it inside a method (debug), that doesn't sound good.
I would then rather keep things as they are.